data/data/powervs/cluster/bootstrap/main.tf

```diff
@@ -0,0 +1,77 @@
+# TODO(mjturek): network and image data blocks can be in main module
+#                as master and bootstrap will be using the same
+#                network and image. Once we add in master module, make
+#                the move.
```

Comment:
This comment is no longer relevant, we can drop it.

Reply:
that's also dropped in #5780